packages/node/src/indexer/indexer.manager.ts

```diff
@@ -157,6 +158,15 @@ export class IndexerManager implements OnApplicationBootstrap {
       sandbox: {
         store: this.storeService.getStore(),
         api: this.apiService.getApi(),
+        getProjectEntry: () => {
+          const pkgPath = path.join(this.nodeConfig.subquery, 'package.json');
```

Comment:
I think we better do this in the host context, e.g when we process project manifest file. 
If we do it in sandbox it will be called every time a block is feed to the mapping function in sandbox.

Reply:
changed to lookup entry point when init the vm.